dspace/config/registries/schema-person-types.xml

```diff
@@ -135,5 +135,22 @@
         <qualifier>isni</qualifier>
         <scope_note>International Standard Name Identifier</scope_note>
     </dc-type>
+    
+    <dc-type>
+        <schema>person</schema>
+        <element>country</element>
+    </dc-type>
+    
+    <dc-type>
+        <schema>person</schema>
+        <element>name</element>
+        <qualifier>translated</qualifier>
+    </dc-type>
+    
+    <dc-type>
+        <schema>person</schema>
+        <element>name</element>
+        <qualifier>variant</qualifier>
+    </dc-type>
```

Comment:
Please add scope notes to all fields.  These two `person.name` fields are especially confusing as we have `person.familyName` and `person.givenName`... so it's unclear whether `person.name.translated` should be a combined name (I assume so).